app/assets/stylesheets/modules/_family.scss

```diff
@@ -131,3 +127,66 @@
   }
 }
 
+.family-names {
+  text-align: center;
+  background-color: tint(gray, 60);
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.